docs/concepts/ale/ale-driver-development-concepts.md

```diff
@@ -0,0 +1,41 @@
+# ALE Driver Development Concepts
+
+## Drivers
+
+- While there are similarities between spacecraft, each instrument has particularities in:
+  - what data it collects/records
+  - how that data is shown
+
+- Drivers allow data from unique and differing spacecraft to be processed into a standard format.
+
```

Comment:
when writing this one out, finishing with a link to the "how to create an ale driver" docs seems like a good idea.